advocacy_docs/partner_docs/CohesityDataProtectforPostgreSQL/05-UsingCohesityDataProtectforPostgreSQL.mdx

```diff
@@ -0,0 +1,182 @@
+
+### Restore an EDB Postgres Advanced Server Instance from a Cohesity Protection Run
+
+You can restore an EDB Postgres Advanced Server instance to the same EDB Postgres Advanced Server or to a different server. At the moment the restores are full database restores, granular restores are not supported yet. 
+
+Recovery Script Arguments are an important piece of the recovery process, they define script parameters for your EDB Postgres Advanced Server recovery. The restore script argument is: `- -target-dir=` with the second half being the path on the target cluster where you want to restore the data.
+
+### EDB Postgres Advanced Server Database Prerequisites Before Recovery Initiation
+
+Before you can recover your EDB Postgres Advanced Server instance, you will need to make sure that the server is not running. Verify that your server is stopped, if it is running, stop it.
+
+**For Example:** Log in to your machine as a Root user, and with systemctl commands first check the status of your instance to determine whether it is up, `systemctl status edb-as-(version)`, and if it is running use `systemctl stop edb-as-(version)` to stop your server for the recovery to take place.
+
+   ![Cohesity Restore Database Steps](Images/CohesityRestoreDatabaseSteps.png)
+
+### Cohesity Recovery of EDB Postgres Advanced Server Data
+
+1. In the Cohesity Dashboard, select `Data Protection` then `Recoveries`.
+
+2. Click `Recover` at the top right of the page and then select `Universal Data Adapter` from the list of sources.
+
+3. Search for the Protection Group that contains the snapshots to recover from by either entering characters of the Protection Group name or entering * for the wildcard option.
+
+   ![Cohesity Recover Choose Protection Group](Images/CohesityRecoverChooseProtectionGroup.png)
+
+   ![Cohesity Recover Protection Group](Images/CohesityRecoverProtectionGroup.png)
+
+4. Select the Protection Group from the search results that you wish to recover.
+
+5. In the Object field name specify a name for the server to be restored.  This name does not affect the restore.  It is used by Cohesity to help track your backups and restores. 
+   By default, the latest snapshot is selected to recover from. If you want to select a point-in-time recovery or a specific data backup that you took, you can click on the pencil to edit the recovery point.
+
+   ![Cohesity Recover Object Name](Images/CohesityRecoverObjectName.png)
+
+   ![Cohesity Recovery Point](Images/CohesityRecoverRecoveryPoint.png)
+
+   !!! Note Note from Cohesity: ”Only cluster level restore is supported and object-level restore is not yet supported, hence the given object name will be ignored by PostgreSQL restore workflow.”
+   !!!
+
+6. Click `Next: Recovery Options`
+
+7. Select `Replace Original` or `Recover to a New Object` depending on your requirement.
+
+8. In the Recovery Options area, you will configure the following options:
+
```

Comment:
why are the bullets spaced?  we don't do this in other areas.

Reply:
We were having a line break issue that was a technical thing with the Markdown code. DeeDee and I went over what was causing it and I let Jasmer know in case we need to add this to any documentation for formatting.